docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -32,11 +31,57 @@ However, you can go right to the completed example.
 Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
 
 The solution is located in the `spring-di-quickstart` {quickstarts-tree-url}/spring-di-quickstart[directory].
+endif::[]
+
+// Tech preview info for community purposes, different from product
+ifndef::product-docs[]
+:extension-status: preview
+endif::[]
+
+//tech preview info for product docs
+ifdef::product-docs[]
+NOTE: The `quarkus-spring-di` extension is available as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] with {ProductLongName}.
+endif::[]
+
+// community-only note about the community suport status of this extension
+ifndef::product-docs[]
+include::./status-include.adoc[]
+endif::[]
+
+ifdef::product-docs[]
+[IMPORTANT]
+====
+The Spring compatibility layer that {ProductName} provides does not start a Spring Application Context or execute any infrastructure classes that are provided by Spring (for example, `org.springframework.beans.factory.config.BeanPostProcessor`) when you run your application.
+{ProductName} can only read metadata from Spring classes and annotations and can parse user code method return types and parameter types that are provided by Spring.
+However, when you add arbitrary libraries that are part of Spring to your {ProductName} application, {ProductName} is not able to use them.
+====
+endif::[]
 
-== Creating the Maven project
+.Prerequisites
 
-First, we need a new project. Create a new project with the following command:
+// comment out the following prerequisite in the alternative 1 layout of the document.
+* You have a {ProductName} Maven project.
+* You have OpenJDK 8 installed.
+* You have Maven {maven-version} installed.
 
+// Alternative 1: generate project form scratch
+////
+[id="creating-the-spring-di-example-maven-project_{context}"]
+== Creating the Spring DI Example Maven project
+
+[role="_abstract"]
+You can create new a new Maven project for the Spring DI example application on the command line.
+
+.Procedure
+
+. Execute the following commands to:
```

Comment:
The following list needs periods a the end of each bullet and to have the first word capitalized.  Per IBM Style Guide: "If list items comprise only complete sentences, include a period after each sentence. If list items comprise only sentence fragments, do not include any end punctuation."

Reply:
```suggestion
. Run the following commands to:
```

IBM Style Guide says to not use 'execute'